sdk/keyvault/azure-keyvault-securitydomain/tsp-location.yaml

```
@@ -0,0 +1,5 @@
+directory: specification/keyvault/data-plane/Security.KeyVault.SecurityDomain
```

Seems like the path is incorrect based on what I see [here](https://github.com/Azure/azure-rest-api-specs/tree/18c83fbaeb90b9b38f017de8d0b9f1d78ceefa98/specification/keyvault/Security.KeyVault.SecurityDomain).
```suggestion
directory: specification/keyvault/Security.KeyVault.SecurityDomain
```